package.json

```diff
@@ -16,25 +16,33 @@
     "lib/"
   ],
   "scripts": {
-    "build": "browserify -d -r ./index.js:hydrolysis -o hydrolysis.js",
-    "build:watch": "watch 'npm run build' ./lib",
-    "release": "browserify -r ./index.js:hydrolysis -o hydrolysis.js",
+    "init": "typings install",
+    "clean": "rm -rf node_modules/ typings/ lib/ && find src | grep \\.js$ | xargs rm",
+    "build": "tsc && babel -q -d lib src && browserify -d -r ./index.js:hydrolysis -o hydrolysis.js",
+    "build:watch": "watch 'npm run build' ./src",
+    "release": "browserify -r ./index.js:hydrolysis -o hydrolysis.js -t babelify",
     "apidocs": "node_modules/jsdoc-to-markdown/bin/cli.js {index.js,lib/{analyzer,loader/*}.js} > API.md",
-    "test": "jshint index.js lib/ && npm run build && wct && mocha test/test.js"
+    "test": "npm run build && wct && mocha test/test.js",
```

Comment:
Will the plain `tsc` invocation fail on warnings? I'd like the tests to be strict here.